python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -482,6 +484,29 @@ def addMissingKeys(self, dataId, butler, missingKeys=None, calibName=None):
             else:
                 raise RuntimeError("No unique lookup for %s: %s" % (k, v))
 
+    def updateMetadata(self, combined, exposureTime, darkTime=None):
+        """Update the metadata"""
+
+        if darkTime is None:
+            darkTime = exposureTime     # avoid warning messages when using calibration products
+
+        visitInfo = afwImage.makeVisitInfo(exposureTime=exposureTime, darkTime=darkTime)
```

Comment:
Is it worth adding `**kwargs` so that additional values could be set easily? I could imagine that `boresightRotAngle` might be important. Presumably the `observatory` could also be set.